proto/desmos/profiles/v1beta1/query.proto

```diff
@@ -37,23 +37,26 @@ service Query {
     option (google.api.http).get = "/desmos/profiles/v1beta1/params";
   }
 
-  // UserRelationships queries the relationships for the user having the given
-  // address
-  rpc UserRelationships(QueryUserRelationshipsRequest)
-      returns (QueryUserRelationshipsResponse) {
+  // Relationships queries all relationships
+  // It queries the relationships for the given user if the user address is
+  // provided
+  rpc Relationships(QueryRelationshipsRequest)
+      returns (QueryRelationshipsResponse) {
     option (google.api.http).get =
         "/desmos/relationships/v1beta1/relationships/{user}";
   }
 
-  // UserBlocks queries the user blocks for the user having the given address
-  rpc UserBlocks(QueryUserBlocksRequest) returns (QueryUserBlocksResponse) {
+  // Blocks queries all blocks
+  // It queries the blocks for the given user if the user address is provided
+  rpc Blocks(QueryBlocksRequest) returns (QueryBlocksResponse) {
     option (google.api.http).get =
         "/desmos/relationships/v1beta1/blocks/{user}";
   }
 
-  // UserChainLinks queries chain links for the given user
-  rpc UserChainLinks(QueryUserChainLinksRequest)
-      returns (QueryUserChainLinksResponse) {
+  // ChainLinks queries all chain links
+  // It queries the chain links for the given user if the user address is
+  // provided
```

Comment:
```suggestion
  // ChainLinks queries the chain links associated to the given user, if provided.
  // Otherwise it queries all the chain links stored.
```